cloud-functions/src/main/java/com/google/fleetevents/lmfs/models/outputs/EtaAssignedOutputEvent.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.fleetevents.lmfs.models.outputs;
+
+import com.google.fleetevents.common.models.FleetEvent;
+import com.google.fleetevents.common.models.OutputEvent;
+import java.util.Objects;
+
+/** Output event for when an eta is assigned to a task. */
+public class EtaAssignedOutputEvent extends OutputEvent {
```

Comment:
We should probably set the type of the event here?

Reply:
Added new type ETA_ASSIGNED